src/Donors/Repositories/DonorRepository.php

```diff
@@ -33,11 +34,8 @@ class DonorRepository
      * Query Donor By ID
      *
      * @since 2.19.6
-     *
-     * @param int $donorId
-     * @return ModelQueryBuilder
```

Comment:
I think we want to leave this with the generic: `@return ModelQueryBuilder<Donor>`. Same for the other usages.

Reply:
Ah yeah good point, I wasn't sure if that would still work but probably not

Reply:
I added back the generics and did an overhaul of docblock cleanups for the models